### PR TITLE
Fix problems discovered by static analysis

### PR DESCRIPTION
**Pull request by szepeviktor**
@chesio

### CODE REVIEW

File: amt-template-tags.php [hunk 1]
```diff
@@ -75,11 +75,6 @@ function amt_metadata_footer() {
     amt_print_footer_block();
 }
 
-function amt_metadata_review() {
-    // Prints full metadata in review mode. No user level checks here.
-    echo amt_get_metadata_inspect();
```
Comment (szepeviktor) [inline amt-template-tags.php:80]: Unused/undefined

File: amt-template-tags.php [hunk 2]
```diff
@@ -152,9 +147,10 @@ function amt_get_user_image_info( $size='thumbnail', $user_id=null ) {
         $image_info['width'] = $main_size_meta[1];
         $image_info['height'] = $main_size_meta[2];
     } elseif ( ! is_null($image_data['url']) ) {
-        $image_info['url'] = $main_size_meta[0];
-        $image_info['width'] = $main_size_meta[1];
-        $image_info['height'] = $main_size_meta[2];
+        // FIXME list( $width, $height ) = wp_getimagesize( $image_data['url'] );
+        $image_info['url'] = $image_data['url'];
+        $image_info['width'] = $width;
+        $image_info['height'] = $height;
```
Comment (szepeviktor) [inline amt-template-tags.php:153]: Seems like unfinished code

**Comment by chesio** [comment]
Thanks @szepeviktor - I need to review these changes in more detail, I'll get back to it soon.

**Comment by szepeviktor** [comment]
@chesio Friendly ping 🏓

**Comment by chesio** [comment]
Thanks Viktor and sorry for the delay!

**Comment by szepeviktor** [comment]
All right, all right, @chesio. Just send the usual €1 000 000.

**Pull request merged**